package.json

```diff
@@ -31,17 +31,18 @@
   "homepage": "https://github.com/fastify/fastify-cookie#readme",
   "devDependencies": {
     "@types/node": "^13.1.0",
-    "fastify": "^2.11.0",
+    "fastify": "github:thomheymann/fastify#fix-instance-types",
     "pre-commit": "^1.2.2",
     "snazzy": "^8.0.0",
     "standard": "^14.0.2",
     "tap": "^12.6.6",
+    "tsd": "^0.11.0",
     "typescript": "^3.3.1"
   },
   "dependencies": {
     "cookie": "^0.4.0",
     "cookie-signature": "^1.1.0",
-    "fastify-plugin": "^1.4.0"
+    "fastify-plugin": "github:thomheymann/fastify-plugin#next"
```

Comment:
Why are we targeting a fork?

Reply:
Because the changes required haven't been published yet

Reply:
Then this work should happen after a required Fastify release is available. This PR cannot be merged while it targets a fork.

Reply:
Correct, Fastify v3 needs to be published before any plugin that depends on v3 can be published. 

The purpose of this piece of work is to verify that v3 typings are working correctly before publishing Fastify v3. We've already discovered multiple issues with the initial type updates by converting plugins to v3. This PR is part of that effort.